Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent { docker defaultWorker.getConfig("openstack-test") }
+    options {
+        ansiColor('xterm')
+        timestamps()
+        timeout(time: 2, unit: "HOURS")
```

Comment:
I can certainly bump the timeout if you want.  The current timeout of 2 hours is based on the current work load (where each job tends to complete in 1-1.5 hours).